lib/Myriad/RPC/Implementation/Redis.pm

```diff
@@ -95,7 +94,7 @@ async sub listener ($self) {
                     if (my $sub = $self->rpc_map->{$message->rpc}) {
                         $sub->[0]->emit($message);
                     } else {
-                        my $error = Myriad::Exception::RPC::MethodNotFound->new($message->rpc);
+                        my $error = Myriad::Exception::RPC::MethodNotFound->new(reason => $message->rpc);
```

Comment:
shouldn't that be changed to this?
```suggestion
                        my $error = Myriad::RPC->MethodNotFound(reason => $message->rpc);
```

Reply:
No the package will be defined by the `declare_exception` helper.

Reply:
ok got it